Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM node:18-alpine AS base
+
+# Install dependencies only when needed
+FROM base AS deps
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+# Install dependencies
+COPY package.json package-lock.json ./
+RUN npm ci
+
+
+# Rebuild the source code only when needed
+FROM base AS builder
+WORKDIR /app
+COPY --from=deps /app/node_modules ./node_modules
+COPY . .
+
+ENV NEXT_TELEMETRY_DISABLED 1
+
+RUN npm run build
+
+# Production image, copy all the files and run next
+FROM base AS runner
+WORKDIR /app
+
+ARG APP_ENV
+ARG GIGS_PROJECT
+ARG GIGS_API_KEY
+
+ENV APP_ENV=$APP_ENV
```

Comment:
double-checking we don't need the ENV variables before running `npm run build`. I'm pretty sure we don't but I just want to confirm

Reply:
No, they're not affecting the build, just the running app 🙂